EMIP/magicscrolls/EMIPms00469/transkribusTextEMIPms00469.xml

```diff
@@ -6,12 +6,12 @@
             <ab>
                <pb facs="#facs_1" n="1r"/>
                <cb facs="#facs_1_r1" n="a"/>
-               <lb facs="#facs_1_line_1656341948235_22" n="1" break="no"/>በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱ<lb facs="#facs_1_r1l12" n="2" break="no"/>ስ፡ ፩አምላክ፡ ጸሎት፡ በእንተ፡ አው<lb facs="#facs_1_r1l13" n="3" break="no"/>ጸሎት፡ በእንተ፡ አውጽእ፡ አጋንንት፡ ወ<lb facs="#facs_1_r1l14" n="4" break="no"/>ሰይጣናት፡ ዘስሙ፡ ቅዱስ፡ ሚካኤል፡ 
+               <lb facs="#facs_1_line_1656341948235_22" n="1" break="no"/>በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱ<lb facs="#facs_1_r1l12" n="2" break="no"/>ሽስAsk Ralph about this፡ ፩አምላክ፡ ጸሎትMy change፡ በእንተ፡ አው<lb facs="#facs_1_r1l13" n="3" break="no"/>ጸሎት፡ በእንተ፡ አውጽእ፡ አጋንንት፡ ወ<lb facs="#facs_1_r1l14" n="4" break="no"/>ሰይጣናት፡ ዘስሙ፡ ቅዱስ፡ ሚካኤል፡ 

```

Comment:
"ሽስAsk Ralph about this፡ ፩አምላክ፡ ጸሎትMy change፡" These seem to be comments maybe not intended for the final version? Also "My change" below